src/test/java/com/wikia/webdriver/testcases/portableinfoboxtests/PortableInfoboxTests.java

```diff
@@ -11,215 +13,346 @@
+    new ArticlePurger().purgeArticleAsAnon();
 
-    String externalLinkName = info
+    Assertion.assertTrue(infobox.clickRedLinkWithIndex(0).isCreateNewArticleModalVisible());
+  }
+
+  public void verifyInternalLinksRedirecting() {
+    PortableInfobox infobox = new PortableInfobox();
+
+    infobox.open(PageContent.PORTABLE_INFOBOX_01);
+    new ArticlePurger().purgeArticleAsAnon();
+
+    String internalLinkName = infobox
         .open(PageContent.PORTABLE_INFOBOX_01)
-        .getExternalLinkRedirectTitleWithIndex(0);
+        .getInternalLinkRedirectTitle(3);
 
-    String externalUrl = info
-        .clickExternalLinkWithIndex(0)
+    String internalURL = infobox
+        .clickInternalLinkWithIndex(3)
         .getUrlAfterPageIsLoaded();
 
-    info.compareURLAndExternalLink(externalLinkName, externalUrl);
+    Assertion.assertEquals(internalLinkName, internalURL);
+  }
+
+  public void verifyExternalLinksRedirecting() {
+    PortableInfobox infobox = new PortableInfobox();
+
+    infobox.open(PageContent.PORTABLE_INFOBOX_01);
+    new ArticlePurger().purgeArticleAsAnon();
 
-    String internalLinkName = info
+    String externalLinkName = infobox
         .open(PageContent.PORTABLE_INFOBOX_01)
-        .getInternalLinkRedirectTitleWithIndex(3);
+        .getExternalLinkRedirectTitle(0);
 
-    String internalURL = info
-        .clickInternalLinkWithIndex(3)
+    String externalUrl = infobox
+        .clickExternalLinkWithIndex(0)
         .getUrlAfterPageIsLoaded();
 
-    info.compareURLAndInternalLink(internalLinkName, internalURL);
+    Assertion.assertEquals(externalLinkName, externalUrl);
   }
 
-  @Execute(onWikia = "mediawiki119")
   public void verifyImagesInWhatLinksHerePage() {
     ArticlePageObject article = new ArticlePageObject();
-    String articleName = article.open(PageContent.PORTABLE_INFOBOX_01).getArticleName();
 
-    article.openSpecialWhatLinksHere(wikiURL).clickPageInputField()
-        .typeInfoboxImageName(PageContent.FILE_IMAGE_NAME).clickShowbutton()
-        .verifyInfoboxArticleInList(articleName);
+    article.open(PageContent.PORTABLE_INFOBOX_01);
+    new ArticlePurger().purgeArticleAsAnon();
+
+    String articleName = article.getArticleName();
+
+    String whatLinkHereResult = article
+        .openSpecialWhatLinksHere(wikiURL)
+        .clickPageInputField()
+        .typeInfoboxImageName(PageContent.FILE_IMAGE_NAME)
+        .clickShowButton()
+        .getWhatLinksHereArticleName(0);
+
+    Assertion.assertTrue(whatLinkHereResult.contains(articleName));
   }
 
-  @Execute(onWikia = "mediawiki119")
   public void verifyLightboxVisibilityAfterClickingImage() {
-    new PortableInfobox().open(PageContent.PORTABLE_INFOBOX_01).clickImage()
-        .isLightboxPresented();
+    PortableInfobox infobox = new PortableInfobox();
+
+    infobox.open(PageContent.PORTABLE_INFOBOX_01);
+    new ArticlePurger().purgeArticleAsAnon();
+
+    Assertion.assertTrue(infobox.clickImage().isLightboxVisible());;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)